src/hooks/index.ts

```diff
@@ -20,26 +18,23 @@ type OnSignup = (eventId : number, orgId : number) => void;
 type OnUndoSignup = (eventId : number, orgId : number) => void;
 
 type EventResponses = {
-    eventResponses: ZetkinEventResponse[] | undefined;
     onSignup: OnSignup;
     onUndoSignup: OnUndoSignup;
 }
 
-export const useEventResponses = () : EventResponses => {
-    const responseQuery = useQuery('eventResponses', getEventResponses());
-    const eventResponses = responseQuery.data;
+export const useEventResponses = (key : string) : EventResponses => {
```

Comment:
The name of this argument is not very clear as to what it will be used for. I see that it's used to decide which queries to invalidate, but that wasn't clear to me before.

I'm wondering if it's even necessary? Shouldn't all queries that include events be invalidated of sign-up state changes?

Reply:
As it is set up at the moment the invalidation is decided upon the key of a query, and since the different event queries have different keys I'm not sue how all of them could be invalidated. Maybe if they all have the same key?